src/util.js

```diff
@@ -31,53 +34,70 @@ class Util {
-        invertedUIObj.notificationBoard(this.msg, 'error');
-        return false;
-      }
+    if (typeof this.allFileUploads[this.file.name] === 'object') {
+      this.msg = 'File(s) already Exist!';
+      invertedUIObj.notificationBoard(this.msg, 'error');
+      return false;
     }
     if (this.file.type !== 'application/json') {
-      this.msg = 'File MUST be JSON!';
+      this.msg = 'File(s) MUST be JSON!';
       invertedUIObj.notificationBoard(this.msg, 'error');
       return false;
     }
     if (this.file.size === 0) {
-      this.msg = 'File cannot be EMPTY!';
+      this.msg = 'File(s) cannot be EMPTY!';
       invertedUIObj.notificationBoard(this.msg, 'error');
       return false;
     }
     invertedUIObj.fileReader(this.file, userEvent);
   }
+
   /**
    * Function searchIndexTest to
    * @param {string} searchQuery
-   * @returns {null}
+   * @returns {Void} Passes result to another function
    */
   searchIndexTest(searchQuery) {
     const selectedFile = invertedUIObj.getSelectedFileToSearch();
     const functionCallName = 'search';
-    if (typeof this.allFileUploads === 'undefined' || this.allFileUploads[selectedFile] === 'undefined' || !this.file.name || typeof this.file === 'undefined') {
+    if (typeof this.allFileUploads === 'undefined' ||
+      this.allFileUploads[selectedFile] === 'undefined' || !this.file.name ||
+      typeof this.file === 'undefined') {
       this.msg = 'You have to firstly Create Index!';
       invertedUIObj.notificationBoard(this.msg, 'error');
     } else if (typeof this.file.name === 'undefined') {
       this.msg = 'Empty Input Detected!';
       invertedUIObj.notificationBoard(this.msg, 'error');
     } else {
-      if (selectedFile === 'all') {
-        this.searchResult = invertedClassObj.searchIndex(searchQuery, selectedFile);
-        invertedUIObj.displayToViewAllSearch(this.searchResult, functionCallName);
-      }
       if (selectedFile !== null && selectedFile !== 'all') {
         this.searchResult = invertedClassObj.searchIndex(searchQuery,
-          selectedFile);
-        invertedUIObj.displayToView(this.searchResult, functionCallName, selectedFile);
-      }
-      if (!selectedFile) {
-        this.searchResult = invertedClassObj.searchIndex(searchQuery, this.file.name);
-        invertedUIObj.displayToView(this.searchResult, functionCallName, this.file.name);
+        selectedFile);
+        invertedUIObj.displayToView(this.searchResult,
+        functionCallName, selectedFile);
+      } else {
+        this.searchResult = invertedClassObj.searchIndex(searchQuery,
+        selectedFile);
+        invertedUIObj.displayToViewAllSearch(this.searchResult,
+        functionCallName);
       }
     }
   }
+
+  /**
+   * validate
+   * This checks if a json object has title and text key
+   * @param {Object} book
+   * @returns {Boolean}
+   */
+  static validate(book) {
+    let ifKeyExist = false;
+    book.forEach((doc) => {
+      if (doc.title && doc.text) {
```

Comment:
You could assign `ifKeyExist` directly to the conditional statement to make this cleaner and shorter, i.e. `lines 95 - 99` can easily be replaced with
`ifKeyExist = (doc.title && doc.text);`